src/js/brim/ingest/getParams.js

```diff
@@ -18,6 +20,7 @@ export type IngestParamsError = {
 export default function getParams(
   data: FileListData,
   dataDir?: string,
+  existingNames?: string[] = [],
```

Comment:
Not sure if you need the `?` optional specifier when you already use an initializer

Also, we might want to convert these parameters to be keys in a single object instead now that we've hit 4 params

Reply:
Yeah, good instincts. I think 4 is close to max for this. I think you do need the optional parameter for flow to let you not provide a value. I could be wrong though.